instruments/instruments.go

```diff
@@ -6,6 +6,10 @@ const Path = "instruments"
 
 type InstrumentType string
 
+const Validation = "/validation/bank-accounts/"
+
+type ValidationType string
```

Comment:
Is this been used? If not then remove it

Reply:
👍